lib/index.js

```diff
@@ -15,31 +15,28 @@ function inspect(root, targetFile, options) {
   if (!options) {
     options = {dev: false};
   }
-  return subProcess.execute(
-    'mvn',
-    buildArgs(root, targetFile, options.args),
-    {cwd: root}
-  )
+  var mvnArgs = buildArgs(root, targetFile, options.args);
+  return subProcess.execute('mvn', mvnArgs, {cwd: root})
     .then(function (result) {
-      var parseResult = {};
       try {
-        parseResult = parse(result, options.dev);
-      } catch (error) {
-        console.log('\nAn unknown error occurred. ' +
-      'Please include the trace below when reporting to Snyk:', error, '\n');
-        return Promise.reject('');
-      }
-      if (parseResult.ok) {
+        var parseResult = parse(result, options.dev);
         return {
           plugin: {
             name: 'bundled:maven',
             runtime: 'unknown',
           },
           package: parseResult.data,
         };
+      } catch (error) {
+        error.message = error.message + '\n\n' +
+          'Please make sure that Apache Maven Dependency Plugin ' +
+          'version 2.2 or above is installed, and that ' +
+          '`mvn ' + mvnArgs.join(' ') + '` executes successfully ' +
+          'on this project.\n\n' +
+          'If the problem persists, collect the output of ' +
+          '`mvn ' + mvnArgs.join(' ') + '` and contact support@snyk.io\n';
+        throw error;
```

Comment:
Whatever error occurred during parsing, here we extend the message to help the user get more insight.